tests/acceptance/features/lib/UsersPage.php

```diff
@@ -658,6 +661,44 @@ public function changeUserPassword(Session $session, $user, $password) {
 		}
 	}
 
+	/**
+	 * @param Session $session
+	 * @param string $user
+	 * @param string $password
+	 *
+	 * @return void
+	 */
+	public function changeUserEmail(Session $session, $user, $password) {
+		$userTr = $this->findUserInTable($user);
+		$editEmailBtn = $userTr->find("xpath", $this->editEmailBtnXpath);
+		if ($editEmailBtn === null) {
```

Comment:
use `$this->assertElementNotNull()`

Reply:
There are quite a few places like this in the methods above. @individual-it should we refactor that stuff while cut-pasting here for this new method?